low-level/files/opt/seagate/sspl/setup/manifest_support_bundle

```diff
@@ -255,9 +345,25 @@ class ManifestSupportBundle(object):
                 encl_dict.update({key:value})
         return encl_dict
 
+    def manifest_bundle_cleanup(self):
+        """Cleanup activity for Manifest Support Bundle"""
+        try:
+            remove_files = [self.encl_manifest, self.node_manifest, self.manifest_summary]
+            for rm_file in remove_files:
+                if os.path.exists(rm_file):
+                    os.remove(rm_file)
+        except Exception as ex:
+            print(f"Failed in manifest bundle cleanup. Error:{str(ex)}")
+
 if __name__ == "__main__":
     if not os.path.isfile(onGoingRequestPath):
         if len(sys.argv) == 3:
+            # Get 'singleton_realstorencl' instance only if sspl_state is active.
+            sspl_state=open(f"/var/{PRODUCT_FAMILY}/sspl/data/state.txt").readline().rstrip().split("=")
```

Comment:
Already we did this code somewhere in the above functions. this time it will duplicate the code.
can we import this module in the build_encl_hw_specifics_instances function? and whenever that module required in another function then we can send an argument like below.
https://github.com/Seagate/cortx-monitor/blob/ca4810c519f2bc0f82259ef8625955bea7658301/low-level/files/opt/seagate/sspl/setup/manifest_support_bundle#L188

        response = self.fetch_ws_request_data("/show/configuration", 'configuration', singleton_realstorencl)

Reply:
@satish-seagate This is not duplicating the code.
I have restructured this piece of code, now we get the singleton realstor instance only if sspl_state is active.
and can use the instance anywhere in the class rather than everytime parsing it as an argument.

Reply:
Okay got it.
can we add some exception handling around this code as we did for early/old code. also need to know that SSPL state is active or not with messages as we did in the old code. don't change the logic.

Reply:
`try:
                sspl_state=open(f"/var/{PRODUCT_FAMILY}/sspl/data/state.txt").readline().rstrip().split("=")
                # Capturing enclosure data only when sspl state is active
                sspl_state = sspl_state[1] if len(sspl_state)==2 else ''
                if sspl_state == 'active':
                    import here
                else:
                    print("SSPL state is not active, So enclosure data are ignored to collecte.")
except (FileNotFoundError, OSError) as e:
                print(f"Failed to open the SSPL 'state.txt' file with an error '{e}',\
                    Can't determine the SSPL state, So enclosure data also being collected.")
                sspl_state = 'active'
                import here